src/components/calcite-tile-select/calcite-tile-select.tsx

```diff
@@ -107,9 +129,10 @@ export class CalciteTileSelect {
 
   @Listen("click")
   click(event: MouseEvent): void {
-    if ((event.target as HTMLElement).localName === "calcite-tile-select") {
+    const target = event.target as HTMLElement;
+    const targets = ["calcite-tile", "calcite-tile-select"];
+    if (targets.includes(target.localName)) {
       this.input.click();
-      this.input.focus();
```

Comment:
Clicking the input automatically focuses it, so the extra call to `.focus()` is redundant.